### PR TITLE
Document container tasks in GMP doc

### PR DESCRIPTION
**Pull request by mattmundell**
**Checklist**:

- [ ] Tests N/A
- [x] [CHANGELOG](https://github.com/greenbone/gvmd/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: src/schema_formats/XML/GMP.xml.in [hunk 1]
```diff
@@ -5777,6 +5777,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
       <p>
         The client uses the create_task command to create a new task.
       </p>
+      <p>
+        When given a target with an id of 0, the command creates a
+        "container" task.  This kind of task can not be run, but it
```
Comment (bjoernricks) [inline src/schema_formats/XML/GMP.xml.in]:
```suggestion
        "container" task. This kind of task can not be run, but it
```

File: src/schema_formats/XML/GMP.xml.in [hunk 2]
```diff
@@ -5777,6 +5777,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
...
+        is possible to import reports into the task.  Importing is
+        done using the create_report command.  The report being
```
Comment (bjoernricks) [inline src/schema_formats/XML/GMP.xml.in]:
```suggestion
        done using the create_report command. The report being
```

File: src/schema_formats/XML/GMP.xml.in [hunk 3]
```diff
@@ -20408,6 +20436,12 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
         the values of the sort order, sort field and apply overrides flag
         that the manager applied when selecting the tasks.
       </p>
+      <p>
+        A task may be a "container" task.  This means the task may not be
```
Comment (bjoernricks) [inline src/schema_formats/XML/GMP.xml.in]:
```suggestion
        A task may be a "container" task. This means the task may not be
```

File: src/schema_formats/XML/GMP.xml.in [hunk 4]
```diff
@@ -20408,6 +20436,12 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
...
+        run, but reports may be uploaded to the task with the command
+        create_report.  Container tasks are identified by having an empty
```
Comment (bjoernricks) [inline src/schema_formats/XML/GMP.xml.in]:
```suggestion
        create_report. Container tasks are identified by having an empty
```

File: src/schema_formats/XML/GMP.xml.in [hunk 5]
```diff
@@ -20899,6 +20933,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
               <name>id</name>
               <type>uuid</type>
               <required>1</required>
+              <summary>ID of target.  Empty for container tasks</summary>
```
Comment (bjoernricks) [inline src/schema_formats/XML/GMP.xml.in]:
```suggestion
              <summary>ID of target. Empty for container tasks</summary>
```

**Comment by mattmundell** [comment]
> Just removed some additional spaces. Not sure if they have been added by intention.

These are to make the text easier to read in fixed width fonts, like on a terminal.
It's intentional (actually automatic for me, it's a very old habit).

I'm reluctant to change it in this patch because it's all over the document already.
Even the licence boilerplate has it (because it's the GNU standard).

**Comment by bjoernricks** [comment]
Feel free to leave the PR as it is :-)

**Pull request merged**